python/tank/platform/engine.py

```diff
@@ -130,8 +130,8 @@ def __init__(self, tk, context, engine_instance_name, env):
         # check general debug log setting and update the global debug flag accordingly. Do not set this
         # flag only when the debug_logging is "true" because the global_debug flag is... global... and it
```

Comment:
this comment seems out of the date with the code now. Or i am not fully understanding its meaning (lol, did i write it?). Maybe we could tweak it to make it clearer?